projects/1.18/assets/ars-elemental/ars_elemental/lang/zh_cn.json

```
@@ -0,0 +1,147 @@
+  "item.ars_elemental.water_bangle": "水之手环",
+  "item.ars_elemental.air_bangle": "气之手环",
+  "item.ars_elemental.earth_bangle": "土之手环",
+  "item.ars_elemental.fire_caster_tome": "火之施法者卷册",
+  "item.ars_elemental.water_caster_tome": "水之施法者卷册",
+  "item.ars_elemental.air_caster_tome": "气之施法者卷册",
+  "item.ars_elemental.earth_caster_tome": "土之施法者卷册",
+  "item.ars_elemental.anima_caster_tome": "灵魂施法者卷册",
+  "tooltip.ars_elemental.caster_tome": "模仿该学派核心的能力",
+  "item.ars_elemental.siren_shards": "塞壬碎片",
+  "item.ars_elemental.siren_charm": "塞壬护符",
+  "item.ars_elemental.firenando_charm": "烈焰加农护符",
+  "item.ars_elemental.curio_bag": "饰品袋",
+  "item.ars_elemental.debug": "调试核心",
+  "block.ars_elemental.water_upstream": "翻腾之心",
+  "block.ars_elemental.mermaid_rock": "塞壬神坛",
+  "block.ars_elemental.yellow_archwood": "闪耀至高木木头",
+  "block.ars_elemental.yellow_archwood_log": "闪耀至高木原木",
+  "block.ars_elemental.stripped_yellow_archwood": "去皮闪耀至高木原木",
+  "block.ars_elemental.stripped_yellow_archwood_log": "去皮闪耀至高木原木",
+  "block.ars_elemental.yellow_archwood_sapling": "闪耀至高木树苗",
+  "block.ars_elemental.yellow_archwood_leaves": "闪耀至高木树叶",
+  "block.ars_elemental.spore_blossom_up": "孢子花（地面）",
+  "block.ars_elemental.fire_turret": "火之灌注炮台",
+  "block.ars_elemental.water_turret": "水之灌注炮台",
+  "block.ars_elemental.air_turret": "气之灌注炮台",
+  "block.ars_elemental.earth_turret": "土之灌注炮台",
+  "entity.ars_elemental.summon_skelehorse": "被召唤的骷髅马",
+  "entity.ars_elemental.summon_direwolf": "被召唤的冰原狼",
+  "entity.ars_elemental.summon_vhex": "被召唤的邪恶恼鬼",
+  "entity.ars_elemental.summon_wskeleton": "被召唤的复仇者",
+  "entity.ars_elemental.summon_dolphin": "被召唤的海豚",
+  "entity.ars_elemental.summon_strider": "被召唤的炽足兽",
+  "entity.ars_elemental.summon_mk": "Summoned Emmekiappo",
+  "entity.ars_elemental.linger_magnet": "引力场",
+  "entity.ars_elemental.siren_entity": "塞壬",
+  "entity.ars_elemental.siren_familiar": "塞壬",
+  "entity.ars_elemental.firenando_entity": "烈焰加农",
+  "entity.ars_elemental.firenando_familiar": "烈焰加农",
+  "entity.ars_elemental.fire_mage": "漂泊火之法师",
+  "entity.ars_elemental.water_mage": "漂泊水之法师",
+  "entity.ars_elemental.air_mage": "漂泊气之法师",
+  "entity.ars_elemental.earth_mage": "漂泊土之法师",
+  "entity.ars_elemental.flashing_weald_walker": "闪耀林地行者",
+  "key.ars_elemental.open_pouch": "[元素魔艺] 打开小饰品袋",
+  "curios.modifiers.an_focus": "当放置在法术核心槽时:",
+  "curios.modifiers.bangle": "当放置在手环槽时:",
+  "curios.identifier.bundle": "手袋",
+  "curios.identifier.bangle": "手环",
+  "effect.ars_elemental.watery_grave": "沉重之水",
+  "effect.ars_elemental.watery_grave.description": "使目标更快沉入水底并加速氧气消耗。",
+  "effect.ars_elemental.hellfire": "地狱之火",
+  "effect.ars_elemental.hellfire.description": "地狱之火可以让下界生物燃烧，降低其治疗效果，并使其更容易被火焰伤害。",
+  "effect.ars_elemental.life_link": "生命链接",
+  "effect.ars_elemental.life_link.description": "施法者将会把自身受到的伤害共享给目标，目标会把受到的治疗效果共享给施法者。",
+  "effect.ars_elemental.enthralled": "魂不着体",
+  "effect.ars_elemental.enthralled.description": "使目标生物被迷惑，令其不会攻击施法者，而是为施法者战斗。",

```

```suggestion
  "effect.ars_elemental.enthralled.description": "使目标生物被魅惑，令其不会攻击施法者，而是为施法者战斗。",
```
[enthral](https://dictionary.cambridge.org/dictionary/english/enthral)
`迷惑`虽然从语义上没有问题，但是少了一份`被勾引`、`被诱惑`的感觉